lib/lookip.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const maxmind = require('maxmind');
+const dbfile = `${__dirname}/../db/GeoLite2-City.mmdb`;
+
+// maxmind singleton
+let _maxdb;
+function maxdb() {
+  if (_maxdb) {
+    return _maxdb;
+  } else {
+    return _maxdb = new Promise((resolve, reject) => {
+      maxmind.open(dbfile, (err, lookup) => {
+        if (err) {
+          reject(err);
+        } else {
+          console.log('--loaded');
+          resolve(lookup);
+        }
+      });
+    });
+  }
+}
+
+// sanity check ip addresses
+function cleanIp(ip) {
+  return (ip || '').split(',').map(s => s.trim())
```

Comment:
getting the first ip if there is a comma separated list of them? like in forwarding headers?